src/main/java/com/jagrosh/jmusicbot/commands/dj/QueueTypeCmd.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 John Grosh <john.a.grosh@gmail.com>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.settings.QueueType;
+import com.jagrosh.jmusicbot.settings.Settings;
+
+/**
+ *
+ * @author Wolfgang Schwendtbauer
+ */
+public class QueueTypeCmd extends DJCommand
+{
+    public QueueTypeCmd(Bot bot)
+    {
+        super(bot);
+        this.name = "queuetype";
+        this.help = "changes the queue type";
+        this.arguments = "[" + QueueType.getNames() + "]";
+        this.aliases = bot.getConfig().getAliases(this.name);
+        this.guildOnly = true;
+    }
+
+    @Override
+    protected void execute(CommandEvent event)
+    {
+        String args = event.getArgs();
+        QueueType value;
+        Settings settings = event.getClient().getSettingsFor(event.getGuild());
+
+        if (args.isEmpty())
+        {
+            if (settings.getQueueType() == QueueType.FAIR)
+                value = QueueType.LINEAR;
+            else
+                value = QueueType.FAIR;
+        }
+        else if(args.equalsIgnoreCase("linear"))
+        {
+            value = QueueType.LINEAR;
+        }
+        else if(args.equalsIgnoreCase("fair"))
+        {
+            value = QueueType.FAIR;
+        }
```

Comment:
I suppose this is fine for now. If we are adding any more queue types, we might want a more dynamic approach.